src/common/displayFormatters.js

```diff
@@ -23,3 +23,15 @@ export const formatPhoneNumber = phone => {
 export const formatAddressLine = (addresses = []) => {
   return addresses.filter(x => x).join(', ');
 };
+
+/**
+ * Title cases string
+ * @param {String} string
+ * @returns {String}
+ */
+export const titleCase = (string = '') => {
+  return string.replace(
+    /\w\S*/g,
+    text => text.charAt(0).toUpperCase() + text.substr(1).toLowerCase(),
+  );
+};
```

Comment:
lodash's `startCase`? :)))

Reply:
let's leave this one as is. 
startCase would incorrectly remove hyphens from some terms eg. 'anti-inflammatory'